### DISCUSSION TITLE
No owner group

### DISCUSSION THREAD

**Discussion by nikitamatrosoff** [Pryaxis/TShock#1937] [General]
TShock _4.4.0 Pre-release 8_ doesn't create group 'owner' automatically but the guide after _/setup_ still mentions it.
![owner](https://user-images.githubusercontent.com/20443664/83124813-244f6680-a0f0-11ea-8f55-1e21123e2750.png)

**Comment by Olink** [comment]
If you already have a Group table, then the default groups are not created.  Can you verify if you already had a Group table in your database?  Otherwise, please backup your tshock.sqlite file somewhere and try again.

**Comment by nikitamatrosoff** [comment]
Of course, I mean the first launch of TShock. Sorry that didn't mention that properly.
